README.md

```diff
@@ -329,7 +353,7 @@ docker run -it --rm -v "$(pwd)/chart":/apps/chart helmunittest/helm-unittest:3.1
 | `postgresql.enabled`                                      | Whether to deploy a PostgreSQL cluster                     | `true`                                                      |
 | `postgresql.backups.enabled`                              | Whether to enable database backups                         | `false`                                                     |
 | `postgresql.cluster.imageName`                            | Name of the PostgreSQL container image                     | `ghcr.io/cloudnative-pg/postgresql:14.17-standard-bookworm` |
-| `postgresql.cluster.instances`                            | Number of PostgreSQL instances                             | `3`                                                         |
+| `postgresql.cluster.instances`                            | Number of PostgreSQL instances                             | `1`                                                         |
```

Comment:
the [default](https://github.com/cloudnative-pg/charts/blob/e66ee1435d1b8a1aac07be106cfafebb9d2afd4c/charts/cluster/values.yaml#L181) in the CPNG chart is 3, so might make sense to copy that. Or does setting 1 make it easier for testing dev setup with fewer nodes?

Reply:
Yes, `1` for local dev (I was struggling for resources). On the `dev` cluster we can make it 3.

Reply:
ah yeah that makes sense